static/src/javascripts/projects/common/modules/experiments/ab.js

```diff
@@ -27,7 +28,8 @@ define([
     Giraffe,
```

Comment:
Yeah, I'd be happy to change 'giraffe'-> 'contributions', though this PR is only to add the new header, so shall I just fix the `giraffeHeader` for the time being?